WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -692,18 +694,11 @@ - (void)refreshNoResultsView
         return;
     }
 
-    if (self.noResultsViewController.view.window) {
-        // The view is already visible.  Nothing more to do.
-        [self adjustNoResultViewPlacement];
-        return;
-    }
-
```

Comment:
I _think_ removing this is causing the NRV to be misplaced sometimes. On a fresh install on an iPhone, going to a Comment filter with no Comments the NRV is too low:

<kbd>![misplaced](https://user-images.githubusercontent.com/1816888/145277024-bbb94b14-41a8-4772-a59e-0d4667e10e72.png)</kbd>

Going back to My Site and selecting Comments again, the NRV is now placed correctly:

<kbd>![correct](https://user-images.githubusercontent.com/1816888/145277094-e69142af-0efe-4d65-87cf-3a8f9ce60c05.png)</kbd>

FWIW, for testing I removed the `[self adjustNoResultViewPlacement]` call on line 700. That fixed it on iPhone, but then caused it to be misplaced on iPad.